model/GamutCompress.nk

```diff
@@ -915,12 +343,11 @@ push $N1c3d9b80
   xpos -40
   ypos 230
  }
-set N5a1f5800 [stack 0]
+set N702b7000 [stack 0]
  Merge2 {
   inputs 2
   operation screen
```

Comment:
One other small thing: Using a screen here is causing negative value in the image to be clamped to 0.0 when plot overlays is enabled.

Using a Plus or an Over instead would fix the issue.